src/plugins/dashboard/public/dashboard_top_nav/internal_dashboard_top_nav.tsx

```diff
@@ -360,7 +366,10 @@ export function InternalDashboardTopNav({
       {viewMode === ViewMode.EDIT ? (
         <DashboardEditingToolbar isDisabled={!!focusedPanelId} />
       ) : null}
-      {showBorderBottom && <EuiHorizontalRule margin="none" />}
+      {showBorderBottom && <EuiHorizontalRule id="dashboardTopNavHorizontalRule" margin="none" />}
+      {isCourierIgnoreFilterEnabled && allDataViews.length > 1 && filters.length > 0 && (
+        <IgnoreFilterTour dataViews={allDataViews} />
```

Comment:
Do you need to pass `dataViews` to `IgnoreFilterTour`? `IgnoreFilterTour` is only added to the DOM when `allDataViews.length > 1`, so why does `IgnoreFilterTour` have a similar check?